src/core_plugins/i18n/server/__tests__/i18n.js

```diff
@@ -0,0 +1,169 @@
+import expect from 'expect.js';
+import * as i18n from '../i18n/i18n';
+import path from 'path';
+import _ from 'lodash';
+
+const FIXTURES = path.join(__dirname, 'fixtures');
+
+describe('i18n module', function () {
+  before('registerTranslations - one plugin', function () {
+    const pluginName = 'test_plugin_1';
+    const pluginTranslationPath = path.join(FIXTURES, 'translations', pluginName);
+    const translationFiles = [
+      path.join(pluginTranslationPath, 'de.json'),
+      path.join(pluginTranslationPath, 'en.json')
+    ];
+    registerTranslations(translationFiles);
+  });
+  describe('one plugin', function () {
+    describe('getTranslationsForLocale', function () {
+      it('en translations are registered' , function () {
+        const locale = 'en';
+        const expectedTranslationJson = {
+          'test_plugin_1-NO_SSL': 'Dont run the dev server using HTTPS',
+          'test_plugin_1-DEV': 'Run the server with development mode defaults',
+          'test_plugin_1-NO_RUN_SERVER': 'Dont run the dev server',
+          'test_plugin_1-HOME': 'Run along home now!'
+        };
+        checkTranslations(locale, expectedTranslationJson);
```

Comment:
Mocha works with promises when you return them from a test function, which is currently happening implicitly. It works fine, but it increases the chances that another developer might come along and accidentally modify the return value of the function and nullify the checks that are happening in the promise. Adding an explicit return makes this less likely because it communicates your intent and the compiler can throw a warning if there's unreachable code after the return.